packages/gatsby/cache-dir/develop-static-entry.js

```diff
@@ -99,9 +214,14 @@ export default (pagePath, callback) => {
     pathname: pagePath,
   })
 
+  let bodyStr = ``
+  if (process.env.GATSBY_EXPERIMENTAL_DEV_SSR && pagePath !== `/dev-404/`) {
+    bodyStr = bodyHtml
+  }
```

Comment:
This function is doing all this work potentially for no reason. Can't this `bodyHtml` be generated conditionally, in its own function, rather than always and conditionally (implicitly) discarded?

Or am I missing another use case for `bodyHtml` and `bodyStr` in this function? Maybe the diff is misleading me.

Reply:
If it's a "no" on the above comment, then condition should be inverted and this ought to just set `bodyHtml = ''` because that's basically what happens here. (Using `bodyHtml` below, dropping `bodyStr` entirely)

Reply:
Yeah, `bodyStr` isn't needed. I'll remove that.

This odd code is temporary as we'll rip out the env flag pretty soon.